### DISCUSSION TITLE
Query LogTime, update Gravity BLocklist, Settings in Webfrontend

### DISCUSSION THREAD

**Discussion by gruppenrichtlinien** [pi-hole/docker-pi-hole#1054] [General]
Hi,
the issue starts with Docker Tag 2022.04, actually running 2022.04.2beta, Pi-hole [v5.9.1],FTL [v5.14] Web Interface [v5.11]

Issues: 

1.  Query LogTime is always: 1970-01-01 01:00:06 -> -e TZ="Europe/Berlin"
while FTL started in webfrontend is showing the right time Time FTL started: | Sun Apr 3 15:22:47 2022 Europe/Berlin

2. update Gravity BLocklist not working -> docker exec pihole pihole updateGravity
[i] Target: https://blocklistproject.github.io/Lists/tracking.txt
[i] Status: Pending...time.c:132: Operation not permitted
timer.c:619: fatal error: RUNTIME_CHECK(isc_time_now((&now)) == 0) failed
[✗] Status: Connection Refused
[✗] List download failed: no cached list available

3. Settings in Webfrontend are not saved, e.G. changing DNS Servers. The webfrontend reports success but checkboxes are resettet.

I completly removed pihole (docker rmi * -f) and start from scratch, running this script from https://github.com/pi-hole/docker-pi-hole/blob/master/docker_run.sh

DNS in resolv.conf is functional, all Clients have internet access using pi-hole as a DNS.

**Comment by dschaper** [comment]
What is your host OS? If it's Buster then this is a known bug with Buster.
